com.unity.perception/Runtime/Randomization/Samplers/SamplerTypes/NormalSampler.cs

```diff
@@ -87,8 +50,9 @@ public void IterateState(int offsetIndex)
         /// <returns>The generated sample</returns>
         public float Sample()
         {
+            var rng = new Unity.Mathematics.Random(ScenarioBase.activeScenario.NextRandomSeed());
```

Comment:
'NextRandomSeed()' makes this method sound like it is doing something different than what it is doing. I suggest renaming this method so that more closely reflects what is really happening and perhaps, updated surrounding documentation

Reply:
Replaced with NextRandomState() to better reflect that the next random value in a sequence of random states has been generated.